modules/server/src/main/java/org/jboss/wsf/stack/cxf/configuration/BusHolder.java

```diff
@@ -398,8 +398,7 @@ protected static void setCXFManagement(Bus bus, Map<String, String> props) {
             InstrumentationManagerExtImpl instrumentationManagerImpl = new InstrumentationManagerExtImpl();
             instrumentationManagerImpl.setBus(bus);
             instrumentationManagerImpl.setEnabled(true);
-            instrumentationManagerImpl.initMBeanServer();
-            instrumentationManagerImpl.register();
```

Comment:
Hi Jim, 
Thanks for reviewing this. I have added, 

            instrumentationManagerImpl.init();
            instrumentationManagerImpl.initMBeanServer();

should i go ahead and commit ? Is this the change you were referring ?